From fab8b4c62034a9987cee2601f49972e597cab50f Mon Sep 17 00:00:00 2001
From: Amit Shah <amit.shah@redhat.com>
Date: Fri, 11 Jul 2014 10:09:08 -0500
Subject: [CHANGE 12/29] tests: vmstate static checker: change description name
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <a1281566ef3d44f657e6c9369bb3ab1b230b1bc0.1405072585.git.amit.shah@redhat.com>
Patchwork-id: 59792
O-Subject: [RHEL7.1 qemu-kvm PATCH 11/18] tests: vmstate static checker: change description name
Bugzilla: 1118707
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

Signed-off-by: Amit Shah <amit.shah@redhat.com>
Reviewed-by: Juan Quintela <quintela@redhat.com>
Signed-off-by: Juan Quintela <quintela@redhat.com>
(cherry picked from commit 1d681c712af592287750fc9ac9f7f75005e7b039)
Signed-off-by: Amit Shah <amit.shah@redhat.com>
Signed-off-by: jen <jen@redhat.com>
---
 tests/vmstate-static-checker-data/dump2.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/vmstate-static-checker-data/dump2.json b/tests/vmstate-static-checker-data/dump2.json
index 34bfbf6..f69966d 100644
--- a/tests/vmstate-static-checker-data/dump2.json
+++ b/tests/vmstate-static-checker-data/dump2.json
@@ -749,7 +749,7 @@
     "version_id": 1,
     "minimum_version_id": 1,
     "Description": {
-      "name": "tpci200",
+      "name": "tpci2002",
       "version_id": 1,
       "minimum_version_id": 1,
       "Fields": [
-- 
1.9.3

